app/gui/src/dashboard/hooks/projectHooks.ts

```diff
@@ -264,7 +264,7 @@ export function useOpenProjectMutation() {
       }))
     },
     onSuccess: async (_, { title, hybrid, suppressHybridProjectOpen = false }) => {
-      await client.cancelQueries({ queryKey: ['project'] })
+      await client.invalidateQueries({ queryKey: ['project'] })
```

Comment:
If this is invalidated here in `onSuccess` it shouldn't also be listed in `meta.invalidates`

Reply:
... oh wait